doc/sphinx/language/core/records.rst

```diff
@@ -161,14 +171,14 @@ Constructing records
          Admitted.
 
          (* Record form: top and bottom can be inferred from other fields *)
-         Definition half' :=
+         Definition half :=
            {| negative := false;
               Rat_bottom_nonzero := O_S 1;
               Rat_irreducible := one_two_irred |}.
 
          (* Applicative form: use the constructor and provide values for all the fields
-            in order *)
-         Definition half := mkRat true 1 2 (O_S 1) one_two_irred.
+            in order.  "mkRat" is the Record command *)
```

Comment:
You mean `Record constructor` I believe??

Reply:
Typo, I left out "defined by".